src/backend/drm/surface/gbm.rs

```diff
@@ -385,7 +392,7 @@ where
 
 /// Errors thrown by a [`GbmBufferedSurface`]
 #[derive(Debug, thiserror::Error)]
-pub enum Error {
+pub enum Error<A: std::error::Error + Send + Sync + 'static> {
```

Comment:
Same here, using `A` as an associated type representing an error feels wrong. We can go multi-letters, like `AllocError` or something like that to still make it clear what this type represents, maybe?